ttnn/cpp/ttnn/operations/pool/downsample/downsample_pybind.hpp

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <optional>
+
+#include "downsample.hpp"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+namespace py = pybind11;
```

Comment:
please see what we do in ccl/all_gather, you can eliminate all this heavy includes

Reply:
done.